cmd/dbseed/resolver_wrappers.go

```diff
@@ -112,13 +112,14 @@ func (s *Seeder) addPlanCollaborator(
 
 	collaborator, _, err := resolvers.CreatePlanCollaborator(
 		context.Background(),
+		s.Config.Store,
+		s.Config.Store,
```

Comment:
Is this intended to be added twice?

Reply:
[EASI-3657](https://jiraent.cms.gov/browse/EASI-3657) Is the ticket I refer to that will resolve this.